main.go

```diff
@@ -8,7 +8,20 @@ import (
 	"github.com/deis/object-storage-cli/config"
 )
 
+var (
+	debugStrs = []string{"DEIS_DEBUG", "DEBUG"}
```

Comment:
I don't think we should check for both DEBUG and DEIS_DEBUG. Since object-storage-cli is present in slugrunner and fetches the slug on boot, if a user specifies `deis config:set DEBUG=1` they'll suddenly get the log message in their application log as shown [in this issue](https://github.com/deis/controller/issues/674).